<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(9,36,43,255)</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../Logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="name_label1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);</string>
   </property>
   <property name="text">
    <string>HTTPS</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_Label2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>40</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TESTING</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scan_Button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;


</string>
   </property>
   <property name="text">
    <string>SCAN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_Text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgba(80,218,107,255);
color: rgb(255, 255, 255);
text-align: center;
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter a URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="target_Label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TARGET</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_mail">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>620</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 5px solid rgba(80,218,107,255);
color: rgb(255, 255, 255);
text-align: center;
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Enter e-mail</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_Button">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>620</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;


</string>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="QLabel" name="send_label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>630</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SEND FULL REPORT TO E-MAIL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white;
color: rgb(255, 255, 255);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;
border-color: rgb(255, 255, 255);

</string>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_Label">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>90</y>
     <width>541</width>
     <height>511</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(27,79,96,70%);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="output_summary_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>591</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(27,79,96,70%);
color: rgb(0, 0, 0);
border-top-left-radius :20px;
border-top-right-radius :20px; 
border-bottom-left-radius : 20px; 
border-bottom-right-radius : 20px;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="HSTS_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>HSTS Preload:</string>
   </property>
  </widget>
  <widget class="QLabel" name="HSTS_Result">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba( 255, 255, 255, 0% );
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="warning_Label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba( 255, 255, 255, 0% );
color:rgba(255,89,41,255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="warning_mail">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>670</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba( 255, 255, 255, 0% );
color:rgba(255,89,41,255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="vulner_Label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>320</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba( 255, 255, 255, 0% );
color:rgba(80,218,107,255);</string>
   </property>
   <property name="text">
    <string>Testing Vulneravility:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cer_Label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba( 255, 255, 255, 0% );
color:rgba(80,218,107,255);</string>
   </property>
   <property name="text">
    <string>Certificate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="protocol_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>21</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(80,218,107,255);
background-color: rgba( 255, 255, 255, 0% );</string>
   </property>
   <property name="text">
    <string>Testing Protocol:</string>
   </property>
  </widget>
  <widget class="QLabel" name="protocol_Result">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>440</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba( 255, 255, 255, 0% );
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>--</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="Cer_Result">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba( 255, 255, 255, 0% );
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="vulner_Result">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>320</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Condensed</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba( 255, 255, 255, 0% );
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
